src/main/java/com/amihaiemil/docker/RtContainers.java

```diff
@@ -158,4 +158,22 @@ HttpClient client() {
     URI baseUri() {
         return this.baseUri;
     }
+
+    /**
+     * Get this container.<br><br>
+     *
+     * @param containerId Id of the Container
+     * @return This container object.
+     */
+    @Override
+    public Container get(final String containerId) {
+        return new RtContainer(
+            Json.createObjectBuilder().build(),
```

Comment:
@jackpan123 can you add the id to this JSON?

```java
Json.createObjectBuilder()
        .add("Id", containerId)
        .build()
```

Reply:
Yes, I will.